File: Class/C语言程序设计实例教程/Chapter_02/test_01.c
#include <stdio.h>

int main(void)
{
    double num=3.4;
    printf("转换成整型后:%d\n",(int)num);
    printf("转换成长整型后:%ld\n",(long int)num);
    printf("转换成单精度输出:%f\n",(float)num);
    return 0;
}

File: Python/Intro_to_Programming_in_Python/Chapter4/4.2/test_01.py
import sys
import stdio

def search(front,later):
    if (later-front)==1:
        return front
    num=(front+later)//2
    stdio.write('Greater than or equal to '+str(num)+'?')
    if stdio.readBool():
        return search(num,later)
    else:
        return search(front,num)
k=int(sys.argv[1])
m=int(sys.argv[2])
n=m**k
stdio.write('Think of a number ')
stdio.writeln('between 0 and '+str(n-1))
guess=search(0,n-1)
stdio.writeln('Your number is '+str(guess))
